apps/web/pages/org/[orgSlug]/[user]/[type].ts

```diff
@@ -1 +1,62 @@
-export { getServerSideProps, default } from "../../../[user]/[type]";
```

Comment:
Earlier we were just going to [user]/[type] when infact it can be a team slug. This was the reason for 404 for Team events.

Now, we identify if the slug is of team or event.